Sources/WheelPicker/CollectionPickerView.swift

```diff
@@ -86,6 +86,7 @@ where Value: Comparable {
 
     init(values: [Value],
          selected: Value,
+         collectionViewBounces: Bool? = true,
```

Comment:
Is there a usecase for passing in `nil` here. As we have a default value, we can probably remove the optional, right?